lib/config/presets/internal/monorepo.ts

```diff
@@ -155,6 +155,7 @@ const repoGroups = {
     'https://github.com/zeit/next.js', // old repo
     'https://github.com/vercel/next.js',
   ],
+  nextra: 'https://github.com/shuding/nextra',
```

Comment:
```suggestion
  nextra-js: 'https://github.com/shuding/nextra',
```

I think we should suffix all new monorepos. @rarkins WDYT?

Reply:
Is there a non-JS version?

Reply:
not yet

Reply:
I think we can keep it as nextra